.swiftpm/xcode/xcuserdata/apdev.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
not sure if this file should be committed... looks like it is specific to your user on your mac